packages/synapse-react-client/src/components/Plot/UpsetPlot.tsx

```diff
@@ -53,14 +59,77 @@ export function UpsetPlot({
   summaryLinkText,
   summaryLink,
   onClick,
+  customPlotColor,
+  selectionOpacity,
 }: UpsetPlotProps) {
   const { accessToken } = useSynapseContext()
   const [isLoading, setIsLoading] = useState<boolean>()
   const [data, setData] = useState<UpsetPlotData>()
   const [error, setError] = useState<string>()
   const [selection, setSelection] = useState(null as ISetLike<any> | null)
+  const theme = useTheme()
+
+  const variantStyles: Record<string, SxProps> = {
+    default: {
+      'text[class^="sChartTextStyle-upset-"], text[class^= "cChartTextStyle-upset-"]':
+        {
+          textTransform: 'uppercase',
+        },
+    },
+    ampals: {
+      width: '100%',
+      '& .selectionHint-upset-': {
+        stroke: 'transparent !important',
+      },
+      '& [class^="selectionHint-upset-"]': {
```

Comment:
Note that these selectors (class names) may change in another release of the UpsetPlot library.